pkg/version/version.go

```diff
@@ -0,0 +1,40 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/submariner-io/admiral/pkg/log"
+)
+
+var (
+	version         = "devel"
+	goRuntimeString = fmt.Sprintf("%s / %s / %s", runtime.Compiler, runtime.Version(), runtime.GOARCH)
+	gitCommit       string
+	gitBuildDate    string
```

Comment:
Names are not consistent:
- all values are strings, so either add "String" suffix to all names, or have no suffix (better)
- gitBuildDate looks wrong, git does not have a concept of a build. Maybe because this is the commit date?
- If we actually show git commit hash and git commit date, better use gitCommitHash and  gitCommitDate
- gitBuildTime is displayed as "Git Commit Time" - either call it gitCommitTime or use "Git Commit Date"
- goRuntimeString value is not consistent with other values, looks like we want 3 separate lines:
  - Go Compiler
  - Go Version
  - Go Arch
  But if most operators use the same format we better use the same format.